index.html

```diff
@@ -1,49 +1,45 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="Style.css">
-    <title>Mobile Portoflio</title>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Mobile Portfolio</title>
+  <link rel="stylesheet" href="Style.css">
 </head>
+
 <body>
-    <div class="container">
-        <header>
-            <h1 class="logo">Ngala</h1>
-            <menu>
-                <ul class="menu">
-                    <li class="mylist"><a href="#">Portoflio</a></li>
-                    <li class="mylist"><a href="#">About</a></li>
-                    <li class="mylist"><a href="#">Contact</a></li>
-                </ul>
-            </menu>
-            <nav>
-                <div class="mybar"></div>
-                <div class="mybar"></div>
-                <div class="mybar"></div>
-                
-            </nav>
-        </header>
-        <section>
+    
+      <header>
+        <h1 class="logo">Ngala</h1>
+        <nav>
+          <div class="mybar"></div>
+          <div class="mybar"></div>
+          <div class="mybar"></div>
+        </nav>
+      </header>
+      <main class="image">
+      <section>
         <div class="background_image">
-            <h1 class="header">I’m Ngala Mac
-                <br>
-                Glad to see <span>you!</span></h1>
-                <p class="bio">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
-                <div class="links">
-                    <h3><a href="#">LET’S CONNECT</a></h3>
-                    <ul>
-                        <li><img src="./Images/twitter.png"></li>
-                        <li><img src="./Images/Linkedin icon.png"></li>
-                        <li><img src="./Images/medium.png"></li>
-                        <li><img src="./Images/github.png"></li>
-                        <li><img src="./Images/twitter.png"></li>
-                     </ul>
-                </div>
+            <div class="shift">
+          <h1 class="header">I’m Ngala<br>Glad to see <span>you!</span></h1>
```

Comment:
- [ ] [OPTIONAL] I apologize🙏🙏 for the oversight by the previous reviewer. 

- [ ] Although it was marked as optional, it is crucial to adhere to the requirement of having only one `<h1>` tag per HTML file. This guideline is essential not only for maintaining proper semantic structure but also for optimizing SEO ranking.
- [ ]  Search engines assign significant importance to the content within the `<h1>` tag, as it typically represents the main heading of the page. By having a single `<h1>` tag that accurately reflects the content, you can enhance your website's SEO ranking and increase its visibility in search engine results. 
- [ ] For more details [here](https://www.semrush.com/blog/h1-tag/)